### DISCUSSION TITLE
Can i run it on pc(windows)  also any one who could help would be great

### DISCUSSION THREAD

**Discussion by patelaryan0914** [ggml-org/whisper.cpp#2333] [General]
![image](https://github.com/user-attachments/assets/12a79f00-69b6-4f27-acd4-edba6fc166db)

**Comment by misutoneko** [comment]
Yes I think you're all set.
A good GPU would help of course, but your CPU is quite new and it supports AVX/AVX2.
And 16GB is sufficient even for the large model.

About Windows... well, I guess I can only offer my condolences about that.
Should be doable though, I see Windows listed as a supported platform. See this thread:
https://github.com/ggerganov/whisper.cpp/discussions/85

Best of luck to you.

**Reply by patelaryan0914** [reply]
yeah  thanks for the condolences it worked tried the base model but upgrading it large but using docker for this can you help i want a realtime transcription so i can pass it to llm generate specific answers or question which eventually drive the conversation to get specific answers .

**Reply by misutoneko** [reply]
I see. Can't really give much advice...(I barely know what docker is, lol :)
But, if you got it working for base, there's really no reason why it shouldn't work for bigger models too.
Getting realtime tho, now that's a whole other kettle of fish. I think you'll need a lot beefier setup for that.

16GB probably isn't enough to run the large model with multiple threads, either.
Anecdotally, with 16GB I generally run medium with two threads. If I use large I think I have to settle for one thread.
(Which, naturally, will be quite slow.)

**Reply by patelaryan0914** [reply]
yeah i see dont know why i jumped for large haven't seen the medium one. Also there's a catch real time means not the audio from the microphone instead my server is serving the audio in the form of mulaw chunks with rate 8000Hz lets see how can i figure it out.. trying to converting them to PCM and serve it to model..

**Reply by ytain** [reply]
You could look at this app for Windows, it is somewhat inspired by whisper.cpp
https://github.com/Const-me/Whisper
